src/models/utils/index.ts

```diff
@@ -1,3 +1,6 @@
+import { OfferCreateFlags, OfferCreateFlagsEnum, PaymentChannelClaimFlags, PaymentChannelClaimFlagsEnum, PaymentTransactionFlags, PaymentTransactionFlagsEnum, Transaction, TrustSetFlags, TrustSetFlagsEnum } from "../transactions";
```

Comment:
Run `eslint --fix` on the new code you're adding

Reply:
we'll get to this later - since the linter rules aren't there if i were you omar I would wait actually

Reply:
The linter rules are already in `2.0` courtesy of #1560